### DISCUSSION TITLE
where do I have to put my fullname?

### DISCUSSION THREAD

**Discussion by kyoofus** [community/community#53305] [GitHub Education]
I'm a Korean University student.   My affiliation card doesn't have  English name. I update my name in Korean in user profile in github.  What should I do?

**Comment by samus-aran** [comment]
Hi @kyoofuskim 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 

Let us know how you get on 👍
